assembl/static2/js/app/routes.jsx

```diff
@@ -29,6 +29,24 @@ import TwoColumnsAdmin from './pages/twoColumnsAdmin';
 import TokenVoteAdmin from './pages/tokenVoteAdmin';
 import JoinDiscussion from './pages/joinDiscussion';
 import { routeForRouter } from './utils/routeMap';
+import { getDomElementOffset } from './utils/globalFunctions';
+
+export const hashLinkScroll = () => {
+  const { hash } = window.location;
+  if (hash !== '') {
+    const id = hash.replace('#', '');
+    // Push onto callback queue so it runs after the DOM is updated,
+    // this is required when navigating from a different page so that
+    // the element is rendered on the page before trying to getElementById.
+    setTimeout(() => {
+      const element = document.getElementById(id);
+      if (element) {
+        const offset = getDomElementOffset(element).top - 160;
```

Comment:
Minor: I would make 160px a variables inside of the global variables file and use it here.

Reply:
I'm against adding a constant in a global variable file if it's used only at one place.